UnitySDK/Assets/ML-Agents/Editor/Tests/MLAgentsEditModeTest.cs

```diff
@@ -116,26 +120,43 @@ public void TestAgent()
         }
     }
 
+    [TestFixture]
     public class EditModeTestInitialization
     {
+        [SetUp]
+        public void SetUp()
+        {
+            Academy.Instance.Dispose();
+        }
+
         [Test]
         public void TestAcademy()
         {
-            var acaGo = new GameObject("TestAcademy");
-            acaGo.AddComponent<Academy>();
-            var aca = acaGo.GetComponent<Academy>();
-            Assert.AreEqual(0, aca.GetStepCount());
```

Comment:
Some of these don't make sense anymore, because we never have an uninitialized Academy